include/exadg/functions_and_boundary_conditions/function_cached.h

```diff
@@ -71,16 +73,16 @@ class FunctionCached
   }
 
   void
-  set_data_pointer(std::map<unsigned int, MapVectorIndex> const & global_map_vector_index_in,
-                   std::map<unsigned int, ArrayTensor> const &    map_solution_in)
+  set_data_pointer(std::map<unsigned int, MapVectorIndex> const &      map_map_vector_index_,
```

Comment:
Is `_` instead of `_in` the new way to indicate that the parameter is an input?

Reply:
We have both versions in the code (the version with `_` only was a recommendation given in the PRACE course "Advanced C++ with focus on software engineering").
We could think about fixing the style and enforce it uniformly within ExaDG in the future.